bin/scrape_software_versions.py

```diff
@@ -35,6 +37,8 @@
 results['Preseq'] = '<span style="color:#999999;\">N/A</span>'
 results['deepTools'] = '<span style="color:#999999;\">N/A</span>'
 results['RSeQC'] = '<span style="color:#999999;\">N/A</span>'
+results['DupRadar'] = '<span style="color:#999999;\">N/A</span>'
```

Comment:
Still `DupRadar` on my radar 😆

Reply:
Last fix. Everything else looks great!

Reply:
I should have finally got it right ....... thanks!